### ISSUE TITLE
USB Detection Error: Only when hosted remotely?

### ISSUE THREAD

**Issue by russeree** [the-via/app#221]
As the title says when I try to host an instance of VIA remotely I get this error? This doesn't happen if I am running everything from my localhost. Is it something to do with the insecure connection?

Thanks in advance.

![image](https://github.com/the-via/app/assets/3104223/94feef7d-1a03-4f7c-b95c-3a3d84f574df)

**Comment by igorsantos07** [comment]
Yep, most probably. These newer Web APIs are all HTTPS only - I couldn't easily find a docs page being explicit about it, though.
